Documentation/Gifbuilder/Gifbuilder/Index.rst

```diff
@@ -1,73 +1,75 @@
-.. include:: /Includes.rst.txt
-.. index:: GIFBUILDER
-.. _gifbuilder:
+..  include:: /Includes.rst.txt
+..  index:: GIFBUILDER
+..  _gifbuilder:
 
 ==========
 GIFBUILDER
 ==========
 
 GIFBUILDER is an object type, which is used in many situations for creating
```

Comment:
```suggestion
:typoscript:`GIFBUILDER` is an object type, which is used in many situations for creating
```
Should we mark gifbuilder as TypoScript throughout as that is a word from TypoScript and not from english?

Reply:
Thought also about it, but there are different mentions of GIFBUILDER, sometimes referencing as "GIFBUILDER object". At first glance, I don't know if it is referring to the TypoScript cObject or something else. So I stpped away from marking all the GIFBUILDER occurrences as TypoScript for now.

Reply:
I think it is the cObject, all cObjects are classes under the hood and therefore can also be called object

Reply:
@linawolf Do you know, why sometimes `GIFBUILDER` is used, and once `->GIFBUILDER`? Is it the same? I also saw this with, for example, `->if`  and `if`.

Reply:
I believe one referes to the TYPOSCRIPT Object and one to the corresponding function in PHP? But maybe also no one thought about it?

Reply:
Don't know, leave it as is for now.

Reply:
Ack